package.json

```diff
@@ -37,16 +37,16 @@
   },
   "homepage": "https://github.com/lencioni/consolidated-events#readme",
   "devDependencies": {
-    "babel-core": "^6.22.1",
-    "babel-jest": "^20.0.0",
-    "babel-preset-airbnb": "^2.2.3",
-    "eslint": "^3.15.0",
-    "eslint-config-airbnb-base": "^11.1.0",
-    "eslint-plugin-import": "^2.3.0",
+    "babel-core": "^6.26.3",
+    "babel-jest": "^23.0.1",
+    "babel-preset-airbnb": "^2.5.1",
+    "eslint": "^4.19.1",
+    "eslint-config-airbnb-base": "^12.1.0",
+    "eslint-plugin-import": "^2.12.0",
     "in-publish": "^2.0.0",
-    "jest": "^20.0.0",
-    "jest-wrap": "^1.0.0",
-    "rimraf": "^2.6.1",
+    "jest": "^23.1.0",
```

Comment:
Upgrading jest will probably not be an option since it aggressively drops support for older nodes.

Reply:
Oh interesting. Good to know. I'm okay with not running tests in Node 4 and 5. I'll update the .travis.yml file now. Thanks for the heads up!

Reply:
My suggestion would be to freeze jest, not to drop testing on supported platforms :-(

Reply:
Or, to do like airbnb will be doing, and switch to tape - since that’s the only test framework that has support for all environments.